VTUL/VTDLP-sitecontent
# Updated html files from VTDLP-siteconfig repo

Jira: https://webapps.es.vt.edu/jira/browse/LIBTD-2284

What does this PR do:
This PR updates the html files in this repo from VTDLP-siteconfig. The html files should now match what is currently in VTDLP-siteconfig repo.

How to test?
Once https://github.com/VTUL/dlp-access/pull/155 is merged, the DLP pages which use this html content should remain the same.

Branch is LIBTD-2284

interested parties:
@yinlinchen
